docs/guides/validation/validation_actions/how_to_trigger_email_as_a_validation_action.md

```diff
@@ -66,7 +66,7 @@ Our [guide on how to Validate data by running a Checkpoint](../how_to_validate_d
 
 If successful, you should receive an email that looks like this:
 
-![image](../../../../docs/images/email_example.png)
+![image](../../../images/email_example.png)
```

Comment:
We should probably make this change everywhere that `../docs/` is but keeping this PR scope small.